src/v/kafka/server/handlers/handler.h

```diff
@@ -18,16 +18,25 @@
 
 namespace kafka {
 
+using memory_estimate_fn = size_t(size_t);
```

Comment:
I don't think it's possible because the important use of this type in this file is as a non-type template parameter to the helper templates below. Also, I don't see any advantage? Primary use of `std::function`-alikes it to capture a type-erased function when a plain function pointer won't do (e.g., when state is captured), right?

Reply:
Yes I understand now, i wrote this comment when I saw the line before i finished the review. Resolving